### PR TITLE
Remove unused includes

### PR DESCRIPTION
**Pull request by erikjv**

### CODE REVIEW

**Review by fmoc — CHANGES_REQUESTED** [review]
Looks good, but CI fails, please fix.

File: src/gui/clientproxy.h [hunk 1]
```diff
@@ -20,9 +20,8 @@
 #include <QRunnable>
 #include <QUrl>
 
-#include <csync.h>
 #include "common/utility.h"
-#include "owncloudlib.h"
+#include <csync.h>
```
Comment (TheOneRing) [inline src/gui/clientproxy.h:24]: better not use a system include

File: src/gui/networksettings.cpp [hunk 2]
```diff
@@ -15,16 +15,16 @@
...
 #include "configfile.h"
 #include "folderman.h"
-#include "accountmanager.h"
+#include "theme.h"
 
+#include <QList>
 #include <QNetworkProxy>
 #include <QString>
-#include <QList>
+#include <qevent.h>
```
Comment (TheOneRing) [inline src/gui/networksettings.cpp:27]: please have a look

**Review by TheOneRing — CHANGES_REQUESTED** [review]

File: src/gui/networksettings.cpp (hunk 2 above)
Comment (erikjv) [reply src/gui/networksettings.cpp:27]: This is on purpose, otherwise QtCore's `QEvent.h` is included, and that does not have one of the gui events.

Comment (TheOneRing) [reply src/gui/networksettings.cpp:27]: oO
Then please add a comment

File: src/gui/clientproxy.h (hunk 1 above)
Comment (erikjv) [reply src/gui/clientproxy.h]: done

File: src/gui/networksettings.cpp (hunk 2 above)
Comment (erikjv) [reply src/gui/networksettings.cpp]: done

**Review by TheOneRing — APPROVED** [review]

**Pull request merged**